packages/ui-react-utils/src/passthroughProps.ts

```diff
@@ -74,13 +79,16 @@ function pick(obj: Record<string, unknown>, keys: string[]) {
   return res
 }
 
-function pickProps(
-  props: Record<string, any>,
-  propTypes: Record<string, any>,
+function pickProps<T extends Record<string, any>>(
```

Comment:
can we somehow indicate in these utils, that they return with part of the object that was passed?

Reply:
I've tried to do it, but I was not successful because these utils are highly dynamic, maybe we should give it another go some time later?

Reply:
I bumped into the same issue in other places too :) Let's try to type them once all the components are finalised.